jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -33,6 +33,28 @@
           hint="Select subset region for background calculation."
         />
 
+        <v-row v-if="bg_subset_selected=='Annulus'">
+          <v-text-field
+            label="Annulus inner radius"
+            v-model="bg_annulus_inner_r"
+            :rules="[() => bg_annulus_inner_r>0 || 'Must be positive.']"
+            hint="Background annulus inner radius in pixels"
+            persistent-hint
+          >
+          </v-text-field>
+        </v-row>
+
+        <v-row v-if="bg_subset_selected=='Annulus'">
+          <v-text-field
+            label="Annulus width"
+            v-model="bg_annulus_width"
+            :rules="[() => bg_annulus_width>0 || 'Must be positive.']"
+            hint="Background annulus width in pixels"

```

Comment:
Is there somehow we can clarify (briefly) that this means outer_radius = inner_radius + width?  Without any visual feedback, I actually had to look in the code to make sure, even though I realize it probably would have been labeled outer radius if that was what it actually meant.

Reply:
> this means outer_radius = inner_radius + width? 

Yes. I stared at this so long, I started to assume everyone knows by heart, so thanks for your fresh pair of eyes. I'll explain.